server/server.go

```diff
@@ -77,6 +77,19 @@ type Server struct {
 
 // NewServer creates the pd server with given configuration.
 func NewServer(cfg *Config) (*Server, error) {
+	if err := cfg.validate(); err != nil {
```

Comment:
From [line 80 to 91](https://github.com/pingcap/pd/pull/226/files#diff-91bbeda7eb98a7adc57b9e47e2cf5c2bR91) ?